gax-java/gax/src/main/java/com/google/api/gax/core/GaxProperties.java

```diff
@@ -43,6 +49,9 @@ public class GaxProperties {
   private static final String DEFAULT_VERSION = "";
   private static final String GAX_VERSION = getLibraryVersion(GaxProperties.class, "version.gax");
   private static final String JAVA_VERSION = getRuntimeVersion();
+  private static final String PROTOBUF_VERSION =
+      getBundleVersion(Any.class).orElse(DEFAULT_VERSION);
+  static final String BUNDLE_VERSION_KEY = "Bundle-Version";
```

Comment:
Do you mind adding a comment regarding where do we get this magic string "Bundle-Version"?

Reply:
added it to the getBundleVersion JavaDoc